src/java/install.sh

```diff
@@ -206,15 +206,15 @@ sdk_install() {
     local full_version_check=${5:-".*-[a-z]+"}
     local set_as_default=${6:-"true"}
     if [ "${requested_version}" = "none" ]; then return; fi
-    # Blank will install latest stable version SDKMAN has
-    if [ "${requested_version}" = "latest" ] || [ "${requested_version}" = "lts" ] || [ "${requested_version}" = "default" ]; then
+     # Blank will install latest stable version SDKMAN has
+    if [ "${requested_version}" = "latest" ] || [ "${requested_version}" = "default" ]; then
```

Comment:
Based on the README,

https://github.com/devcontainers/features/blob/9387225ac7ada315d94511848fd490a8519ed591/src/java/README.md?plain=1#L18

This `install.sh` should receive `latest`, if nothing is passed. However, it automatically sets 'lts' as default, which is different from README.

That was mentioned in #1011

Reply:
AH, thank's for pointing this out. It's definitely a bug, @gauravsaini04 can we update to match with the default `latest`? 

[features/src/java/install.sh](https://github.com/devcontainers/features/blob/9387225ac7ada315d94511848fd490a8519ed591/src/java/install.sh#L12)

Reply:
corrected